frontend/src/components/Contexts/MissionFilterContext.tsx

```diff
@@ -41,117 +60,260 @@ const completedStatuses = [
+            },
+            switchMinStartTime: (newMinStartTime: number | undefined) => {
+                if (
+                    (newMinStartTime && filterState.minEndTime && newMinStartTime > filterState.minEndTime) ||
+                    (filterState.maxStartTime && newMinStartTime! > filterState.maxStartTime)
+                )
+                    setFilterError(
+                        TranslateText('minStartTime') +
+                            ' ' +
+                            TranslateText('cannot be greater than') +
+                            ' ' +
+                            TranslateText('minEndTime') +
+                            ' ' +
+                            TranslateText('or') +
+                            TranslateText('maxStartTime')
+                    )
+                else {
+                    setFilterIsSet(true)
+                    setFilterState({ ...filterState, minStartTime: newMinStartTime })
+                }
+            },
+            switchMaxStartTime: (newMaxStartTime: number | undefined) => {
+                if (filterState.maxEndTime && newMaxStartTime && newMaxStartTime > filterState.maxEndTime)
+                    setFilterError(
+                        TranslateText('maxStartTime') +
+                            ' ' +
+                            TranslateText('cannot be greater than') +
+                            ' ' +
+                            TranslateText('maxEndTime')
+                    )
+                else if (filterState.minStartTime && newMaxStartTime && newMaxStartTime < filterState.minStartTime)
+                    setFilterError(
+                        TranslateText('maxStartTime') +
+                            ' ' +
+                            TranslateText('cannot be less than') +
+                            ' ' +
+                            TranslateText('minStartTime')
+                    )
+                else {
+                    setFilterIsSet(true)
+                    setFilterState({ ...filterState, maxStartTime: newMaxStartTime })
+                }
+            },
+            switchMinEndTime: (newMinEndTime: number | undefined) => {
+                if (filterState.maxEndTime && newMinEndTime && newMinEndTime > filterState.maxEndTime)
+                    setFilterError(
+                        TranslateText('minEndTime') +
+                            ' ' +
+                            TranslateText('cannot be greater than') +
+                            ' ' +
+                            TranslateText('maxEndTime')
+                    )
+                else if (filterState.minStartTime && newMinEndTime && newMinEndTime < filterState.minStartTime)
+                    setFilterError(
+                        TranslateText('minEndTime') +
+                            ' ' +
+                            TranslateText('cannot be less than') +
+                            ' ' +
+                            TranslateText('minStartTime')
```

Comment:
Sorry but I'm not sure what you mean by using bold here? Are you suggesting that I make it lower case but that I also make the text bold using Typography? Or to do this instead of using lower case? And I'm not sure what you mean by using ' '?

Reply:
Meant either making it lower case or using ' ' and bold. Making it lower case is a sufficient solution!